isotpsend.c

```diff
@@ -166,7 +168,10 @@ int main(int argc, char **argv)
 		    break;
 
 	    case 't':
-		    opts.frame_txtime = strtoul(optarg, (char **)NULL, 10);
+		    if (!strncmp(optarg, ZERO_STRING, strlen(ZERO_STRING)))
+			    opts.frame_txtime = CAN_ISOTP_FRAME_TXTIME_ZERO;
+		    else
+			    opts.frame_txtime = strtoul(optarg, (char **)NULL, 10);
```

Comment:
nitpick: cast not needd

Reply:
Good hint. I removed all these obsolete casts from the existing tools in a follow-up patch.